src/Uno.UI/ts/Windows/ApplicationModel/Contacts/ContactPicker.ts

```diff
@@ -0,0 +1,50 @@
+﻿enum ContactProperty {
+	Address = "address",
+	Email = "email",
+	Icon = "icon",
+	Name = "name",
+	Tel = "tel"
+};
+
+
+declare class ContactInfo {
+	address: PaymentAddress[];
+	email: string[];
+	name: string;
+	tel: string;
+}
+
+declare class ContactsManager {
+	select(props: ContactProperty[], options: any): Promise<ContactInfo[]>;
+}
+
+interface Navigator {
+	contacts: ContactsManager
+}
+
+namespace Windows.ApplicationModel.Contacts {
+
+	export class ContactPicker {
```

Comment:
Codacy found an issue: [A maximum of 1 class per file is allowed.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6378321)